.github/workflows/full_suite_integration_tests_cron.yml

```diff
@@ -0,0 +1,191 @@
+          echo "**Timestamp:** ${{ steps.current-time.outputs.formattedTime }}" >> message.txt
+          echo "**Job run:** [${{ github.run_id }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})" >> message.txt
+          echo "**Build scan URL:** ${{ needs.run-it-full-suite.outputs.build-scan-url }}" >> message.txt
+          echo "**Commits since the last run:**" >> message.txt
+          list=$(curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/commits?since=${{ steps.last-run-date.outputs.last-run-date }}" | jq ' .[] | .sha' -r)
+          if [ -z "$list" ]
+          then
+            echo "No new commits" >> message.txt
+          else
+            echo "|commit SHA|PR|" >> message.txt
+            echo "|---|:---:|" >> message.txt
+            while IFS= read -r commit; do
+              resp=$(curl -H "Accept: application/vnd.github.groot-preview+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/commits/${commit}/pulls" | jq '.[] | {html_url, title}')
+              if [ -z "$resp" ]; then
+                echo "|${commit}|-|" >> message.txt
+              else
+                echo "|${commit}|[PR-$(echo $resp | jq .title -r)]($(echo $resp | jq .html_url -r))|" >> message.txt
+              fi
+            done <<< "$list"
+          fi
+
+      - name: Create success message
+        if: ${{ needs.run-it-full-suite.outputs.it-status == 'success' }}
+        run: |
+          echo "### Full suite IT run :white_check_mark: SUCCEEDED :white_check_mark:" >> message.txt
+          echo "**Timestamp:** ${{ steps.current-time.outputs.formattedTime }}" >> message.txt
+          echo "**Job run:** [${{ github.run_id }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})" >> message.txt
+          echo "**Build scan URL:** ${{ needs.run-it-full-suite.outputs.build-scan-url }}" >> message.txt
+          echo "**Closing issue**" >> message.txt
+
+      - name: Make message body
+        id: body
+        run: |
+          body=$(cat message.txt)
+          body="${body//'%'/'%25'}"
+          body="${body//$'\n'/'%0A'}"
+          body="${body//$'\r'/'%0D'}"
+          echo "::set-output name=body::${body}"
+
+      - name: Decide on next step
+        id: next-step
+        run: |
+          next_step=''
+          status=${{ needs.run-it-full-suite.outputs.it-status }}
+          issue_number=${{ steps.check-opened.outputs.issue-number }}
+          if [ "$status" == 'success' ]; then
+            if [ "$issue_number" == null ]; then
+              next_step='do-nothing'
+            else
+              next_step='close'
+            fi
+          else
+            if [ "$issue_number" == null ]; then
+              next_step='create'
+            else
+              next_step='comment'
+            fi
+          fi
```

Comment:
if else hell

Reply:
True, yet I was not able to implement it with a bash switch. It will be improved once rewrote to flank-scripts, that's for sure